nrf_security/src/mbedtls/CMakeLists.txt

```diff
@@ -1,20 +1,40 @@
 #
-# Copyright (c) 2019 Nordic Semiconductor
-#
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
-FUNCTION(PREPEND var prefix)
-  SET(listVar "")
+
+#
+# Internal function to append all unnamed parameters with 'prefix'
+# and add it to the list 'var'
+#
+FUNCTION(append_with_prefix var prefix)
```

Comment:
lowercase

Reply:
All CMake keywords that can be lowercase should be lowercase

Reply:
Fixed